frontend/canvas-chrome-ext/webpack.config.js

```diff
@@ -52,7 +52,7 @@ module.exports = {
                 new HtmlWebpackPlugin({
             template: './src/html/compilation-notification.html',
             filename: 'compilation-notification.html',
-            chunks: ['compilationStatus']
+            chunks: ['compilation-notification']
```

Comment:
just curious, what is the chunks field for?

Reply:
It's to have multiple template for html. Since we have a separate html file for compilation-notification, I didn't want to lose that and added it as a fail-safe. It may not be necessary when we refactor to use React format. 
https://yonatankra.com/how-to-use-htmlwebpackplugin-for-multiple-entries/